docs/configuration.md

```diff
@@ -3,7 +3,7 @@
 ## Configuring `haskell-language-server`
 
 Language servers like `haskell-language-server` expose most of their configuration via the client (i.e. the editor).
-That means that the way in which you configure the settings will depend on the client.
+That means that how you configure the settings will depend on the client.
```

Comment:
IMO, "the way in which" is pretty clear any does not need a change

Reply:
Ooh okay, I change it as it was